CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in each version of the haproxy cookbook.
 
 ## [unreleased]
 
+## [v6.2.6](2018-11-05)
+
+- Put use_backend rules before the http-request rules
```

Comment:
It's putting the `http-request` rules before the `use_backend` so this statement should be flipped

Reply:
Thanks for catching this. it was a bit late last night